In [1]:
import pandas as pd

In [2]:
nursing_homes=pd.read_csv('../data/NH_capstone_data_CSV.csv')
nursing_homes

Processing Date                                      Provider Name  \
0            1/1/2025                           BURNS NURSING HOME, INC.   
1            1/1/2025                     COOSA VALLEY HEALTHCARE CENTER   
2            1/1/2025                         HIGHLANDS HEALTH AND REHAB   
3            1/1/2025        EASTVIEW REHABILITATION & HEALTHCARE CENTER   
4            1/1/2025                      PLANTATION MANOR NURSING HOME   
...               ...                                                ...   
14398        1/1/2025  LAS ALTURAS NURSING & TRANSITIONAL CARE BROWNS...   
14399        1/1/2025        RIVERSIDE NURSING AND REHABILITATION CENTER   
14400        1/1/2025  FIVE POINTS NURSING & REHABILITATION OF COLLEG...   
14401        1/1/2025                   HCA HOUSTON HEALTHCARE SOUTHEAST   
14402        1/1/2025                       LEGACY ESTATE LONG TERM CARE   

             City/Town State County/Parish                  Ownership Type  \
0         RUSSELLVILLE    AL      Franklin                      For Profit   
1            SYLACAUGA    AL     Talladega                      For Profit   
2           SCOTTSBORO    AL       Jackson         Government- City/County   
3           BIRMINGHAM    AL     Jefferson                      For Profit   
4             MC CALLA    AL     Jefferson                      For Profit   
...                ...   ...           ...                             ...   
14398      BROWNSVILLE    TX       Cameron                      For Profit   
14399        KERRVILLE    TX          Kerr                      For Profit   
14400  COLLEGE STATION    TX        Brazos                      For Profit   
14401         PASADENA    TX        Harris                      For Profit   
14402         COMANCHE    TX      Comanche  Government - Hospital district   

       Number of Certified Beds  Average Number of Residents per Day  \
0                            57                                 53.5   
1                            85                                 76.4   
2                            50                                 45.7   
3                            92                                 69.3   
4                           103                                 87.6   
...                         ...                                  ...   
14398                        90                                  NaN   
14399                       100                                 31.0   
14400                       116                                  NaN   
14401                         4                                  6.3   
14402                       100                                  NaN   

               Provider Type                             Legal Business Name  \
0      Medicare and Medicaid                        BURNS NURSING HOME, INC.   
1      Medicare and Medicaid              COOSA VALLEY HEALTHCARE CENTER LLC   
2      Medicare and Medicaid                  HH HEALTH SYSTEM - JACKSON LLC   
3      Medicare and Medicaid                    BALL HEALTHCARE EASTVIEW INC   
4      Medicare and Medicaid                 C & G HEALTHCARE SERVICES, INC.   
...                      ...                                             ...   
14398  Medicare and Medicaid     HERITAGE BROWNSVILLE HEALTH DEVELOPMENT LLC   
14399  Medicare and Medicaid             KERRVILLE HEALTHCARE OPERATIONS LLC   
14400  Medicare and Medicaid               Legal Business Name Not Available   
14401               Medicare                                CHCA BAYSHORE LP   
14402  Medicare and Medicaid  COMANCHE COUNTY CONSOLIDATED HOSPITAL DISTRICT   

       ... Rating Cycle 2 Standard Health Survey Date  \
0      ...                                  8/21/2019   
1      ...                                  6/13/2019   
2      ...                                   6/6/2019   
3      ...                                  2/20/2020   
4      ...                                   

In [3]:
nursing_homes=nursing_homes.rename(columns={'County/Parish': 'County','City/Town':'City', 'Qaulity Management (QM) Rating':'Quality Management (QM) Rating'})
#going to grab count of nursing homes per county per state???

In [4]:
#nursing_homes.columns.tolist()

In [5]:
nursing_homes[['Ownership Type']].value_counts()

Ownership Type                
For Profit                        10781
Non profit - Corporation           2438
Government- City/County             438
Non profit - Church related         292
Government - Hospital district      281
Government - State                  157
Government - Federal                 16
Name: count, dtype: int64

In [6]:
state_counts=nursing_homes[['State']].value_counts()
state_counts

State
TX       1174
CA       1150
OH        914
FL        679
IL        670
PA        657
NY        587
MO        480
IN        472
MI        418
NC        411
IA        385
NJ        344
MA        336
MN        323
WI        315
GA        300
TN        299
KS        297
VA        285
OK        282
KY        263
LA        262
AL        221
AR        217
MD        215
CO        208
MS        196
CT        193
WA        190
SC        179
NE        168
AZ        139
OR        125
WV        122
UT         94
SD         88
ID         78
ME         73
NH         72
ND         72
RI         72
NV         67
NM         66
MT         55
DE         43
HI         40
WY         35
VT         33
DC         16
AK         16
PR          6
GU          1
Name: count, dtype: int64

In [7]:
avg_nursing_homes=state_counts.mean()
avg_nursing_homes
#all of nursing home ownership types

271.75471698113205

In [8]:
#wanting to make sure im pulling data with every possible FOR PROFIT combo

In [9]:
state_for_profit=nursing_homes[nursing_homes['Ownership Type'].str.contains('For Profit', case=False, na=False)]
#state_ranking_for_profit

In [10]:
state_count_for_profit=state_for_profit.groupby('State').size().sort_values(ascending=False)
state_count_for_profit

State
CA    1008
TX     847
OH     770
IL     547
FL     537
NY     410
PA     406
MO     384
NC     348
MI     317
NJ     281
IN     263
OK     254
MA     251
TN     246
GA     239
IA     226
KY     215
LA     215
VA     214
AR     197
AL     190
WI     190
KS     187
CO     175
MD     170
MS     158
CT     155
WA     149
SC     144
AZ     122
OR     107
MN     104
WV      96
UT      72
NE      71
ID      63
RI      62
NV      57
ME      56
NM      52
NH      45
SD      32
DE      32
MT      29
VT      24
HI      24
WY      18
ND       7
DC       7
PR       4
AK       4
dtype: int64

In [11]:
avg_state_count_for_profit=state_count_for_profit.mean()
avg_state_count_for_profit

207.32692307692307

In [12]:
#Same steps as above but not for NON PROFIT combos

In [13]:
state_non_profit=nursing_homes[nursing_homes['Ownership Type'].str.contains('Non Profit', case=False, na=False)]

In [14]:
state_count_non_profit=state_non_profit.groupby('State').size().sort_values(ascending=False)
state_count_non_profit

State
PA    228
MN    194
NY    151
IA    150
OH    132
FL    124
TX    115
CA    114
IL    106
KS     89
WI     88
IN     88
MA     83
MO     72
NE     68
MI     64
ND     63
VA     62
NC     55
NJ     55
SD     53
GA     49
MD     42
KY     42
TN     38
LA     37
CT     36
WA     31
OK     23
WV     22
AL     22
SC     20
CO     18
MT     18
NH     17
ME     17
MS     16
OR     15
AZ     12
AR     12
AK     10
RI     10
HI     10
NM      9
DE      9
DC      9
UT      8
VT      8
ID      5
WY      5
NV      4
PR      2
dtype: int64

In [15]:
avg_state_count_non_profit=state_count_non_profit.mean()
avg_state_count_non_profit

52.5

In [16]:
county_state_count=nursing_homes.groupby(['State','County']).size().reset_index(name='Count')
county_state_count
#doesnt help too much, going to look at the top county for each state

State                County  Count
0       AK             Anchorage      3
1       AK                Bethel      1
2       AK               Chugach      1
3       AK  Fairbanks North Star      1
4       AK       Kenai Peninsula      3
...    ...                   ...    ...
2806    WY            Sweetwater      2
2807    WY                 Teton      1
2808    WY                 Uinta      1
2809    WY              Washakie      1
2810    WY                Weston      1

[2811 rows x 3 columns]

In [17]:
max_county=county_state_count.loc[county_state_count.groupby('State')['Count'].idxmax()]
max_county=max_county.sort_values(by='Count', ascending=False)
max_county
#this found us the top county in each state -- will pull in population data and circle back to this

State               County  Count
180     CA          Los Angeles    369
612     IL                 Cook    200
2365    TX               Harris     98
1845    OH             Cuyahoga     93
156     AZ             Maricopa     76
1066    MA            Middlesex     69
1372    MO            St. Louis     68
324     FL             Pinellas     63
1188    MI                Wayne     63
263     CT              Capitol     62
2055    PA           Montgomery     57
1807    NY               Queens     57
1216    MN             Hennepin     53
2650    WA                 King     48
741     IN               Marion     45
1756    NV                Clark     43
1073    MD            Baltimore     42
2085    RI           Providence     41
933     KY            Jefferson     39
1965    OK             Oklahoma     39
47      AL            Jefferson     34
824     KS              Johnson     33
1723    NJ             Monmouth     33
2004    OR            Multnomah     33
2506    UT            Salt Lake     31
2706    WI            Milwaukee     30
1553    NC          Mecklenburg     29
543     IA                 Polk     28
2262    TN               Shelby     26
1012    LA       E. Baton Rouge     25
464     HI             Honolulu     25
274     DE           New Castle     24
135     AR              Pulaski     23
238     CO            Jefferson     23
1651    NE              Douglas     22
1706    NH         Hillsborough     21
1732    NM           Bernalillo     18
384     GA               Fulton     18
272     DC         The District     16
2109    SC           Greenville     16
1097    ME           Cumberland     15
2617    VA  Virginia Beach City     14
566     ID                  Ada     14
1410    MS                Hinds     11
2753    WV              Kanawha     11
2171    SD            Minnehaha     10
1597    ND                 Cass      8
1495    MT          Yellowstone      7
2629    VT           Chittenden      5
2795    WY              Fremont      4
0       AK            Anchorage      3
2080    PR             San Juan      2
462     GU                 Guam      1

In [18]:
#state_rating=nursing_homes.loc[nursing_homes.groupby('State')['Overall Rating'].idxmax()]
#state_rating=state_rating.sort_values(by='Overall Rating', ascending=False)
#state_rating
#this is grabbing the data for the max nursing home rating in each state--I believe it's just the first high score--possibly not important

In [19]:
#state_rating[['State','Ownership Type','Overall Rating']]
#same data as above but only specific columns--I want an average rating and dropping ownership type column 

In [20]:
avg_state_rating=nursing_homes.groupby('State', as_index=False)['Overall Rating'].mean()
avg_state_rating=avg_state_rating.sort_values(by='Overall Rating', ascending=False)
avg_state_rating

State  Overall Rating
0     AK        3.625000
7     DC        3.562500
40    PR        3.500000
12    HI        3.461538
14    ID        3.350649
8     DE        3.214286
3     AZ        3.166667
46    UT        3.129032
24    MN        3.065421
32    NJ        3.064516
4     CA        3.031551
2     AR        3.004630
41    RI        3.000000
29    ND        3.000000
49    WA        3.000000
36    OH        2.996674
31    NH        2.985915
9     FL        2.985141
34    NV        2.984848
16    IN        2.976546
13    IA        2.976316
50    WI        2.971154
23    MI        2.953995
5     CO        2.946860
6     CT        2.906250
27    MT        2.905660
39    PA        2.882263
17    KS        2.867347
21    MD        2.859155
38    OR        2.846774
35    NY        2.837329
1     AL        2.822727
47    VA        2.822064
30    NE        2.796407
20    MA        2.793413
48    VT        2.781250
22    ME        2.763889
42    SC        2.761364
44    TN        2.678082
28    NC        2.628362
33    NM        2.625000
18    KY        2.615385
37    OK        2.596429
52    WY        2.588235
45    TX        2.577453
43    SD        2.534884
51    WV        2.483333
26    MS        2.476923
25    MO        2.463158
15    IL        2.412030
19    LA        2.249042
10    GA        2.246622
11    GU        2.000000

In [21]:
#alaska has the highest avg overall rating- look at population in comparison to georgia with second lowest avg rating

In [22]:
nursing_homes.columns.tolist()

['Processing Date',
 'Provider Name',
 'City',
 'State',
 'County',
 'Ownership Type',
 'Number of Certified Beds',
 'Average Number of Residents per Day',
 'Provider Type',
 'Legal Business Name',
 'Date First Approved to Provide Medicare and Medicaid Services',
 'Affiliated Entity Name',
 'Continuing Care Retirement Community',
 'Abuse Icon',
 'Most Recent Health Inspection More Than 2 Years Ago',
 'Provider Changed Ownership in Last 12 Months',
 'With a Resident and Family Council',
 'Overall Rating',
 'Health Inspection Rating',
 'Quality Management (QM) Rating',
 'Long-Stay QM Rating',
 'Short-Stay QM Rating',
 'Short-Stay QM Rating Footnote',
 'Staffing Rating',
 'Reported Nurse Aide Staffing Hours per Resident per Day',
 'Reported LPN Staffing Hours per Resident per Day',
 'Reported RN Staffing Hours per Resident per Day',
 'Reported Licensed Staffing Hours per Resident per Day',
 'Reported Total Nurse Staffing Hours per Resident per Day',
 'Total number of nurse staff hours per

In [23]:
fines_by_state=nursing_homes.groupby('State')['Total Amount of Fines in Dollars'].sum().reset_index()
fines_by_state=fines_by_state.sort_values(by='Total Amount of Fines in Dollars', ascending=False)
fines_by_state

State  Total Amount of Fines in Dollars
15    IL                       74489506.78
45    TX                       62410474.70
4     CA                       32498809.93
23    MI                       27680621.54
36    OH                       27393335.37
9     FL                       25152844.43
28    NC                       23736647.50
39    PA                       22969259.36
20    MA                       18546218.63
25    MO                       16886931.92
50    WI                       14818340.24
32    NJ                       14341311.47
49    WA                       14012006.86
19    LA                       13474195.22
44    TN                       11573570.65
13    IA                       11466033.84
35    NY                       10459577.60
24    MN                        8759501.85
16    IN                        8696801.67
37    OK                        7926715.58
5     CO                        7596014.54
18    KY                        7209619.72
10    GA                        7025931.42
41    RI                        6988663.48
17    KS                        6791167.70
21    MD                        5764754.56
38    OR                        5714229.78
47    VA                        5671551.69
6     CT                        5239879.35
33    NM                        4790001.16
42    SC                        4748147.00
26    MS                        4023012.40
51    WV                        3594329.27
27    MT                        3114568.93
1     AL                        3003496.72
30    NE                        2888629.06
46    UT                        2870888.97
8     DE                        2385719.12
7     DC                        2354788.43
48    VT                        2183512.24
2     AR                        2148381.19
14    ID                        1647432.86
43    SD                        1339965.05
12    HI                        1337642.13
31    NH                        1177822.01
29    ND                        1055042.49
3     AZ                        1042069.55
34    NV                         795861.15
52    WY                         697023.33
22    ME                         540302.38
0     AK                         473564.56
40    PR                         207612.59
11    GU                              0.00

In [24]:
#fines_by_state['Total Amount of Fines in Dollars']=pd.to_numeric(fines_by_state['Total Amount of Fines in Dollars'], errors='coerce')

#trying to convert to $$$$ but having trouble, moving on

In [25]:
#fines_by_state['Total Amount of Fines in Dollars'].fillna(0, inplace=True)

In [26]:
#fines_by_state['Total Amount of Fines in Dollars']=fines_by_state['Total Amount of Fines in Dollars'].map('${:,.2f}'.format)
#fines_by_state

In [27]:
abuse_counts=nursing_homes.groupby(['State', 'Abuse Icon']).size().reset_index(name='Abuse Counts')

In [28]:
abuse_counts_pivot=abuse_counts.pivot(index='State', values='Abuse Counts', columns='Abuse Icon').reset_index()

In [29]:
abuse_counts_pivot=abuse_counts_pivot.sort_values(by='Y', ascending=False)

In [30]:
abuse_counts_pivot=abuse_counts_pivot.fillna(0)
abuse_counts_pivot
#double checked the count of nursing homes for each state and checks out--will want to remove abuse icon on index??

Abuse Icon State       N      Y
15            IL   472.0  198.0
4             CA  1020.0  130.0
45            TX  1048.0  126.0
39            PA   576.0   81.0
28            NC   344.0   67.0
25            MO   418.0   62.0
36            OH   856.0   58.0
9             FL   628.0   51.0
23            MI   368.0   50.0
44            TN   260.0   39.0
5             CO   170.0   38.0
19            LA   228.0   34.0
35            NY   555.0   32.0
21            MD   187.0   28.0
16            IN   445.0   27.0
17            KS   270.0   27.0
49            WA   164.0   26.0
6             CT   167.0   26.0
38            OR   100.0   25.0
3             AZ   118.0   21.0
51            WV   103.0   19.0
20            MA   319.0   17.0
26            MS   180.0   16.0
13            IA   369.0   16.0
10            GA   284.0   16.0
18            KY   247.0   16.0
47            VA   270.0   15.0
42            SC   164.0   15.0
46            UT    81.0   13.0
32            NJ   331.0   13.0
50            WI   302.0   13.0
43            SD    76.0   12.0
1             AL   209.0   12.0
37            OK   271.0   11.0
33            NM    55.0   11.0
41            RI    61.0   11.0
52            WY    25.0   10.0
14            ID    69.0    9.0
34            NV    59.0    8.0
29            ND    65.0    7.0
27            MT    48.0    7.0
24            MN   316.0    7.0
48            VT    28.0    5.0
2             AR   213.0    4.0
7             DC    12.0    4.0
8             DE    40.0    3.0
22            ME    71.0    2.0
31            NH    71.0    1.0
12            HI    39.0    1.0
0             AK    16.0    0.0
11            GU     1.0    0.0
30            NE   168.0    0.0
40            PR     6.0    0.0

Oberservation- for Nebraska- there's 0 reported abuses which is amazing considering the size of the amount of nursing homes, it's not the avg number for each state but it's not the smallest either.

Illinois has the highest amount of abuses reported of 198

Dive deeper into Illinois because the state also has the highest amount of fines at $74 million, TX is second place with $62

In [31]:
illinois=nursing_homes[nursing_homes['State']=='IL']

look at ownership types, licensed help week vs. weekend, healthcode ranking, resident and family, number of beds and avg residents per day

In [32]:
illinois[["With a Resident and Family Council"]].value_counts()

With a Resident and Family Council
Resident                              598
Both                                   38
Family                                  5
Name: count, dtype: int64

In [33]:
illinois[["Health Inspection Rating"]].value_counts()

Health Inspection Rating
2.0                         191
4.0                         150
1.0                         131
3.0                         128
5.0                          65
Name: count, dtype: int64

In [34]:
illinois[["Quality Management (QM) Rating"]].value_counts()

Quality Management (QM) Rating
2.0                               192
3.0                               166
4.0                               127
1.0                               118
5.0                                58
Name: count, dtype: int64

In [35]:
illinois[["Staffing Rating"]].value_counts()

Staffing Rating
1.0                314
2.0                169
4.0                 70
3.0                 67
5.0                 42
Name: count, dtype: int64

In [36]:
illinois[["Reported Nurse Aide Staffing Hours per Resident per Day"]].mean()

Reported Nurse Aide Staffing Hours per Resident per Day    2.092052
dtype: float64

In [37]:
illinois[["Reported LPN Staffing Hours per Resident per Day"]].mean()

Reported LPN Staffing Hours per Resident per Day    0.630462
dtype: float64

In [38]:
illinois[["Reported RN Staffing Hours per Resident per Day"]].mean()

Reported RN Staffing Hours per Resident per Day    0.70387
dtype: float64

In [39]:
illinois[["Reported Licensed Staffing Hours per Resident per Day"]].mean()

Reported Licensed Staffing Hours per Resident per Day    1.334333
dtype: float64

In [40]:
illinois[["Total number of nurse staff hours per resident per day on the weekend"]].mean()

Total number of nurse staff hours per resident per day on the weekend    2.993575
dtype: float64

In [41]:
illinois[["Reported Physical Therapist Staffing Hours per Resident Per Day"]].mean()

Reported Physical Therapist Staffing Hours per Resident Per Day    0.046722
dtype: float64

In [42]:
ohio=nursing_homes[nursing_homes['State']=='OH']

In [43]:
ohio[["With a Resident and Family Council"]].value_counts()

With a Resident and Family Council
Resident                              718
Both                                  128
Family                                  1
Name: count, dtype: int64

In [44]:
ohio[["Health Inspection Rating"]].value_counts()

Health Inspection Rating
2.0                         222
4.0                         207
3.0                         204
1.0                         176
5.0                          93
Name: count, dtype: int64

In [45]:
ohio[["Quality Management (QM) Rating"]].value_counts()

Quality Management (QM) Rating
5.0                               375
4.0                               280
3.0                               161
2.0                                63
1.0                                14
Name: count, dtype: int64

In [46]:
ohio[["Staffing Rating"]].value_counts()

Staffing Rating
2.0                310
1.0                236
3.0                205
4.0                122
5.0                 29
Name: count, dtype: int64

In [47]:
ohio[["Reported Nurse Aide Staffing Hours per Resident per Day"]].mean()

Reported Nurse Aide Staffing Hours per Resident per Day    2.111601
dtype: float64

In [48]:
ohio[["Reported LPN Staffing Hours per Resident per Day"]].mean()

Reported LPN Staffing Hours per Resident per Day    0.953957
dtype: float64

In [49]:
ohio[["Reported RN Staffing Hours per Resident per Day"]].mean()

Reported RN Staffing Hours per Resident per Day    0.615496
dtype: float64

In [50]:
ohio[["Reported Licensed Staffing Hours per Resident per Day"]].mean()

Reported Licensed Staffing Hours per Resident per Day    1.569453
dtype: float64

In [51]:
ohio[["Reported Physical Therapist Staffing Hours per Resident Per Day"]].mean()

Reported Physical Therapist Staffing Hours per Resident Per Day    0.050731
dtype: float64

In [52]:
nursing_homes

Processing Date                                      Provider Name  \
0            1/1/2025                           BURNS NURSING HOME, INC.   
1            1/1/2025                     COOSA VALLEY HEALTHCARE CENTER   
2            1/1/2025                         HIGHLANDS HEALTH AND REHAB   
3            1/1/2025        EASTVIEW REHABILITATION & HEALTHCARE CENTER   
4            1/1/2025                      PLANTATION MANOR NURSING HOME   
...               ...                                                ...   
14398        1/1/2025  LAS ALTURAS NURSING & TRANSITIONAL CARE BROWNS...   
14399        1/1/2025        RIVERSIDE NURSING AND REHABILITATION CENTER   
14400        1/1/2025  FIVE POINTS NURSING & REHABILITATION OF COLLEG...   
14401        1/1/2025                   HCA HOUSTON HEALTHCARE SOUTHEAST   
14402        1/1/2025                       LEGACY ESTATE LONG TERM CARE   

                  City State     County                  Ownership Type  \
0         RUSSELLVILLE    AL   Franklin                      For Profit   
1            SYLACAUGA    AL  Talladega                      For Profit   
2           SCOTTSBORO    AL    Jackson         Government- City/County   
3           BIRMINGHAM    AL  Jefferson                      For Profit   
4             MC CALLA    AL  Jefferson                      For Profit   
...                ...   ...        ...                             ...   
14398      BROWNSVILLE    TX    Cameron                      For Profit   
14399        KERRVILLE    TX       Kerr                      For Profit   
14400  COLLEGE STATION    TX     Brazos                      For Profit   
14401         PASADENA    TX     Harris                      For Profit   
14402         COMANCHE    TX   Comanche  Government - Hospital district   

       Number of Certified Beds  Average Number of Residents per Day  \
0                            57                                 53.5   
1                            85                                 76.4   
2                            50                                 45.7   
3                            92                                 69.3   
4                           103                                 87.6   
...                         ...                                  ...   
14398                        90                                  NaN   
14399                       100                                 31.0   
14400                       116                                  NaN   
14401                         4                                  6.3   
14402                       100                                  NaN   

               Provider Type                             Legal Business Name  \
0      Medicare and Medicaid                        BURNS NURSING HOME, INC.   
1      Medicare and Medicaid              COOSA VALLEY HEALTHCARE CENTER LLC   
2      Medicare and Medicaid                  HH HEALTH SYSTEM - JACKSON LLC   
3      Medicare and Medicaid                    BALL HEALTHCARE EASTVIEW INC   
4      Medicare and Medicaid                 C & G HEALTHCARE SERVICES, INC.   
...                      ...                                             ...   
14398  Medicare and Medicaid     HERITAGE BROWNSVILLE HEALTH DEVELOPMENT LLC   
14399  Medicare and Medicaid             KERRVILLE HEALTHCARE OPERATIONS LLC   
14400  Medicare and Medicaid               Legal Business Name Not Available   
14401               Medicare                                CHCA BAYSHORE LP   
14402  Medicare and Medicaid  COMANCHE COUNTY CONSOLIDATED HOSPITAL DISTRICT   

       ... Rating Cycle 2 Standard Health Survey Date  \
0      ...                                  8/21/2019   
1      ...                                  6/13/2019   
2      ...                                   6/6/2019   
3      ...                                  2/20/2020   
4      ...                                   5/2/2019   
...    ...              

In [53]:
nursing_homes['Total'] = pd.to_numeric(nursing_homes['Total Amount of Fines in Dollars'], errors='coerce')

In [54]:
nursing_homes['Total'].apply(lambda x: f"${x:,.2f}")
nursing_homes

Processing Date                                      Provider Name  \
0            1/1/2025                           BURNS NURSING HOME, INC.   
1            1/1/2025                     COOSA VALLEY HEALTHCARE CENTER   
2            1/1/2025                         HIGHLANDS HEALTH AND REHAB   
3            1/1/2025        EASTVIEW REHABILITATION & HEALTHCARE CENTER   
4            1/1/2025                      PLANTATION MANOR NURSING HOME   
...               ...                                                ...   
14398        1/1/2025  LAS ALTURAS NURSING & TRANSITIONAL CARE BROWNS...   
14399        1/1/2025        RIVERSIDE NURSING AND REHABILITATION CENTER   
14400        1/1/2025  FIVE POINTS NURSING & REHABILITATION OF COLLEG...   
14401        1/1/2025                   HCA HOUSTON HEALTHCARE SOUTHEAST   
14402        1/1/2025                       LEGACY ESTATE LONG TERM CARE   

                  City State     County                  Ownership Type  \
0         RUSSELLVILLE    AL   Franklin                      For Profit   
1            SYLACAUGA    AL  Talladega                      For Profit   
2           SCOTTSBORO    AL    Jackson         Government- City/County   
3           BIRMINGHAM    AL  Jefferson                      For Profit   
4             MC CALLA    AL  Jefferson                      For Profit   
...                ...   ...        ...                             ...   
14398      BROWNSVILLE    TX    Cameron                      For Profit   
14399        KERRVILLE    TX       Kerr                      For Profit   
14400  COLLEGE STATION    TX     Brazos                      For Profit   
14401         PASADENA    TX     Harris                      For Profit   
14402         COMANCHE    TX   Comanche  Government - Hospital district   

       Number of Certified Beds  Average Number of Residents per Day  \
0                            57                                 53.5   
1                            85                                 76.4   
2                            50                                 45.7   
3                            92                                 69.3   
4                           103                                 87.6   
...                         ...                                  ...   
14398                        90                                  NaN   
14399                       100                                 31.0   
14400                       116                                  NaN   
14401                         4                                  6.3   
14402                       100                                  NaN   

               Provider Type                             Legal Business Name  \
0      Medicare and Medicaid                        BURNS NURSING HOME, INC.   
1      Medicare and Medicaid              COOSA VALLEY HEALTHCARE CENTER LLC   
2      Medicare and Medicaid                  HH HEALTH SYSTEM - JACKSON LLC   
3      Medicare and Medicaid                    BALL HEALTHCARE EASTVIEW INC   
4      Medicare and Medicaid                 C & G HEALTHCARE SERVICES, INC.   
...                      ...                                             ...   
14398  Medicare and Medicaid     HERITAGE BROWNSVILLE HEALTH DEVELOPMENT LLC   
14399  Medicare and Medicaid             KERRVILLE HEALTHCARE OPERATIONS LLC   
14400  Medicare and Medicaid               Legal Business Name Not Available   
14401               Medicare                                CHCA BAYSHORE LP   
14402  Medicare and Medicaid  COMANCHE COUNTY CONSOLIDATED HOSPITAL DISTRICT   

       ... Rating Cycle 2 Total Health Score  \
0      ...                               8.0   
1      ...                               4.0   
2      ...                              20.0   
3      ...                               4.0   
4      ...                               4.0   
...    ...                               ...   
14398  ...                    

In [55]:
nursing_homes[["Affiliated Entity Name"]].value_counts().head(10)

Affiliated Entity Name            
THE ENSIGN GROUP                      287
GENESIS HEALTHCARE                    220
LIFE CARE CENTERS OF AMERICA          194
PACS GROUP                            188
SIMCHA HYMAN & NAFTALI ZANZIPER       129
CREATIVE SOLUTIONS IN HEALTHCARE      128
SABER HEALTHCARE GROUP                126
COMMUNICARE HEALTH                    124
TRILOGY HEALTH SERVICES               121
EVANGELICAL GOOD SAMARITAN SOCIETY     93
Name: count, dtype: int64

In [56]:
ensign=nursing_homes[nursing_homes["Affiliated Entity Name"]=="THE ENSIGN GROUP"]

In [57]:
nursing_homes['Total Amount of Fines in Dollars'].sum()

553714329.97

In [58]:
state_for_profit['Total Amount of Fines in Dollars'].sum()/nursing_homes['Total Amount of Fines in Dollars'].sum()*100

85.32097488710401

In [59]:
state_non_profit['Total Amount of Fines in Dollars'].sum()/nursing_homes['Total Amount of Fines in Dollars'].sum()*100

10.237003140784005

In [60]:
columns_needed=[
'Adjusted Nurse Aide Staffing Hours per Resident per Day',
       'Adjusted LPN Staffing Hours per Resident per Day',
       'Adjusted RN Staffing Hours per Resident per Day',
       'Adjusted Total Nurse Staffing Hours per Resident per Day',
       'Adjusted Weekend Total Nurse Staffing Hours per Resident per Day',
       'Rating Cycle 1 Standard Survey Health Date',
       'Rating Cycle 1 Total Health Score',
       'Rating Cycle 2 Standard Health Survey Date',
       'Rating Cycle 2 Total Health Score',
       'Rating Cycle 3 Standard Health Survey Date',
       'Rating Cycle 3 Total Health Score',
       'Number of Facility Reported Incidents',
       'Number of Substantiated Complaints',
       'Number of Citations from Infection Control Inspections',
       'Number of Fines', 'Total Amount of Fines in Dollars',
       'Total Number of Penalties',
    'Total number of nurse staff hours per resident per day on the weekend',
       'Registered Nurse hours per resident per day on the weekend',
       'Reported Physical Therapist Staffing Hours per Resident Per Day',
       'Total nursing staff turnover', 'Registered Nurse turnover','Staffing Rating','With a Resident and Family Council', 'Overall Rating',
       'Health Inspection Rating', 'Quality Management (QM) Rating','Abuse Icon', 'Affiliated Entity Name','Ownership Type', 'Number of Certified Beds',
       'Average Number of Residents per Day','State']

In [61]:
state_for_profit=state_for_profit[columns_needed]

In [79]:
state_for_profit.columns.tolist()

['Adjusted Nurse Aide Staffing Hours per Resident per Day',
 'Adjusted LPN Staffing Hours per Resident per Day',
 'Adjusted RN Staffing Hours per Resident per Day',
 'Adjusted Total Nurse Staffing Hours per Resident per Day',
 'Adjusted Weekend Total Nurse Staffing Hours per Resident per Day',
 'Rating Cycle 1 Standard Survey Health Date',
 'Rating Cycle 1 Total Health Score',
 'Rating Cycle 2 Standard Health Survey Date',
 'Rating Cycle 2 Total Health Score',
 'Rating Cycle 3 Standard Health Survey Date',
 'Rating Cycle 3 Total Health Score',
 'Number of Facility Reported Incidents',
 'Number of Substantiated Complaints',
 'Number of Citations from Infection Control Inspections',
 'Number of Fines',
 'Total Amount of Fines in Dollars',
 'Total Number of Penalties',
 'Total number of nurse staff hours per resident per day on the weekend',
 'Registered Nurse hours per resident per day on the weekend',
 'Reported Physical Therapist Staffing Hours per Resident Per Day',
 'Total nursing st

In [68]:
#state_for_profit.groupby['State'],['Affiliated Entity Name'].sum()

In [69]:
#common_NH=nursing_homes.groupby(['State'])['Affiliated Entity Name'].unique().reset_index().rename(columns={'State':'States'})

In [70]:
#common_NH

In [84]:
nh_for_profit=nursing_homes[nursing_homes['Ownership Type']=='For Profit']

Processing Date                                      Provider Name  \
0            1/1/2025                           BURNS NURSING HOME, INC.   
1            1/1/2025                     COOSA VALLEY HEALTHCARE CENTER   
3            1/1/2025        EASTVIEW REHABILITATION & HEALTHCARE CENTER   
4            1/1/2025                      PLANTATION MANOR NURSING HOME   
5            1/1/2025               ATHENS HEALTH AND REHABILITATION LLC   
...               ...                                                ...   
14397        1/1/2025                  MIDTOWNE MEADOWS HEALTH AND REHAB   
14398        1/1/2025  LAS ALTURAS NURSING & TRANSITIONAL CARE BROWNS...   
14399        1/1/2025        RIVERSIDE NURSING AND REHABILITATION CENTER   
14400        1/1/2025  FIVE POINTS NURSING & REHABILITATION OF COLLEG...   
14401        1/1/2025                   HCA HOUSTON HEALTHCARE SOUTHEAST   

                  City State     County Ownership Type  \
0         RUSSELLVILLE    AL   Franklin     For Profit   
1            SYLACAUGA    AL  Talladega     For Profit   
3           BIRMINGHAM    AL  Jefferson     For Profit   
4             MC CALLA    AL  Jefferson     For Profit   
5               ATHENS    AL  Limestone     For Profit   
...                ...   ...        ...            ...   
14397       MIDLOTHIAN    TX      Ellis     For Profit   
14398      BROWNSVILLE    TX    Cameron     For Profit   
14399        KERRVILLE    TX       Kerr     For Profit   
14400  COLLEGE STATION    TX     Brazos     For Profit   
14401         PASADENA    TX     Harris     For Profit   

       Number of Certified Beds  Average Number of Residents per Day  \
0                            57                                 53.5   
1                            85                                 76.4   
3                            92                                 69.3   
4                           103                                 87.6   
5                           149                                131.7   
...                         ...                                  ...   
14397                       121                                 72.4   
14398                        90                                  NaN   
14399                       100                                 31.0   
14400                       116                                  NaN   
14401                         4                                  6.3   

               Provider Type                          Legal Business Name  \
0      Medicare and Medicaid                     BURNS NURSING HOME, INC.   
1      Medicare and Medicaid           COOSA VALLEY HEALTHCARE CENTER LLC   
3      Medicare and Medicaid                 BALL HEALTHCARE EASTVIEW INC   
4      Medicare and Medicaid              C & G HEALTHCARE SERVICES, INC.   
5      Medicare and Medicaid         ATHENS HEALTH AND REHABILITATION LLC   
...                      ...                                          ...   
14397  Medicare and Medicaid                     HACO HEALTH MIDTOWNE LLC   
14398  Medicare and Medicaid  HERITAGE BROWNSVILLE HEALTH DEVELOPMENT LLC   
14399  Medicare and Medicaid          KERRVILLE HEALTHCARE OPERATIONS LLC   
14400  Medicare and Medicaid            Legal Business Name Not Available   
14401               Medicare                             CHCA BAYSHORE LP   

       ... Rating Cycle 2 Total Health Score  \
0      ...                               8.0   
1      ...                               4.0   
3      ...                               4.0   
4      ...                               4.0   
5      ...                              20.0   
...    ...                               ...   
14397  ...                               0.0   
14398  ...                               NaN   
14399  ...                               NaN   
14400  ...                               NaN   
14401  ...                               NaN   

      Rating Cycle 3 Standard

In [73]:
state_for_profit['Total Amount of Fines in Dollars'].sum()

472434464.42

In [74]:
state_non_profit['Total Amount of Fines in Dollars'].sum()

56683753.35000001

In [75]:
state_for_profit.groupby(['Ownership Type', 'State'])['Overall Rating'].mean().unstack()

State            AK        AL       AR        AZ        CA        CO  \
Ownership Type                                                         
For Profit      2.5  2.761905  3.02551  3.198347  2.906094  2.873563   

State                 CT        DC        DE        FL  ...        SD  \
Ownership Type                                          ...             
For Profit      2.714286  2.428571  2.870968  2.835206  ...  1.935484   

State                 TN        TX        UT        VA        VT        WA  \
Ownership Type                                                               
For Profit      2.690083  2.406213  3.211268  2.443396  2.565217  2.871622   

State                 WI        WV        WY  
Ownership Type                                
For Profit      2.684492  2.242105  2.235294  

[1 rows x 52 columns]

In [81]:
state_for_profit.groupby(['Ownership Type', 'State'])['Staffing Rating'].mean()

Ownership Type  State
For Profit      AK       4.000000
                AL       3.280423
                AR       2.795918
                AZ       2.677686
                CA       2.777332
                CO       2.781609
                CT       2.915584
                DC       3.857143
                DE       3.419355
                FL       2.926554
                GA       1.727660
                HI       3.833333
                IA       3.179372
                ID       2.935484
                IL       1.682657
                IN       2.022989
                KS       2.756757
                KY       2.327014
                LA       2.163551
                MA       2.534137
                MD       2.553571
                ME       3.745455
                MI       2.913462
                MN       3.313725
                MO       1.902632
                MS       3.496815
                MT       2.392857
                NC       2.284884
                ND       3

In [85]:
state_non_profit.groupby(['State', 'Ownership Type'])['Overall Rating'].mean().unstack()

Ownership Type  Non profit - Church related  Non profit - Corporation
State                                                                
AK                                      NaN                  3.900000
AL                                 2.333333                  2.947368
AR                                      NaN                  2.583333
AZ                                 4.000000                  2.909091
CA                                 4.076923                  3.959596
CO                                 4.000000                  3.562500
CT                                 4.000000                  3.656250
DC                                 5.000000                  4.375000
DE                                      NaN                  4.111111
FL                                 3.750000                  3.456140
GA                                 2.750000                  2.931818
HI                                      NaN                  4.333333
IA                                 3.062500                  3.136364
ID                                      NaN                  3.000000
IL                                 4.160000                  3.587500
IN                                 2.888889                  3.141026
KS                                 3.714286                  3.172840
KY                                 4.000000                  3.184211
LA                                 2.333333                  3.000000
MA                                 3.000000                  3.679487
MD                                 4.166667                  3.800000
ME                                 1.000000                  3.562500
MI                                 4.000000                  3.775862
MN                                 3.066667                  3.432927
MO                                 3.714286                  3.253968
MS                                      NaN                  3.250000
MT                                      NaN                  3.529412
NC                                 3.000000                  4.018519
ND                                 3.636364                  3.039216
NE                                 2.666667                  3.369231
NH                                 4.500000                  3.076923
NJ                                 3.200000                  4.180000
NM                                      NaN                  3.444444
NV                                      NaN                  3.000000
NY                                 2.500000                  3.160839
OH                                 3.208333                  3.611111
OK                                 3.000000                  2.777778
OR                                      NaN                  3.133333
PA                                 3.826087                  3.868293
PR                                      NaN                  3.000000
RI                                 2.000000                  3.888889
SC                                 4.500000                  3.777778
SD                                 4.200000                  2.638298
TN                                 2.333333                  2.657143
TX                                 5.000000                  3.401786
UT                                      NaN                  2.375000
VA                                 4.600000                  3.980392
VT                                      NaN                  3.125000
WA                                 2.666667                  3.407407
WI                                 3.428571                  3.179104
WV                                 4.000000                  3.368421
WY                                      NaN                  3.400000

In [86]:
church_nh=state_non_profit[state_non_profit['Ownership Type']=='Non profit - Church related']
church_nh

Processing Date                                      Provider Name  \
148          1/1/2025                  LAUDERDALE CHRISTIAN NURSING HOME   
190          1/1/2025                           ST MARTIN'S IN THE PINES   
219          1/1/2025  FATHER PURCELL MEMORIAL EXCEPTIONAL CHILDREN'S...   
345          1/1/2025                   SUN HEALTH GRANDVIEW CARE CENTER   
672          1/1/2025                PROVIDENCE ST ELIZABETH CARE CENTER   
...               ...                                                ...   
13398        1/1/2025                      VILLA SCALABRINI SPECIAL CARE   
13453        1/1/2025                               BETHEL LUTHERAN HOME   
13550        1/1/2025                 MCCULLOUGH HALL NURSING CENTER INC   
13621        1/1/2025                         THE METHODIST HOSPITAL SNF   
14185        1/1/2025    THE WESLEYAN SKILLED NURSING AND REHABILITATION   

                  City State       County               Ownership Type  \
148             KILLEN    AL   Lauderdale  Non profit - Church related   
190           IRONDALE    AL    Jefferson  Non profit - Church related   
219         MONTGOMERY    AL   Montgomery  Non profit - Church related   
345      SUN CITY WEST    AZ     Maricopa  Non profit - Church related   
672    NORTH HOLLYWOOD    CA  Los Angeles  Non profit - Church related   
...                ...   ...          ...                          ...   
13398       SUN VALLEY    CA  Los Angeles  Non profit - Church related   
13453            SELMA    CA       Fresno  Non profit - Church related   
13550      SAN ANTONIO    TX        Bexar  Non profit - Church related   
13621          HOUSTON    TX       Harris  Non profit - Church related   
14185       GEORGETOWN    TX   Williamson  Non profit - Church related   

       Number of Certified Beds  Average Number of Residents per Day  \
148                          58                                 56.6   
190                         128                                 80.9   
219                          58                                 43.2   
345                          72                                 63.5   
672                          52                                 43.2   
...                         ...                                  ...   
13398                        58                                 59.3   
13453                        59                                 55.2   
13550                        51                                 16.2   
13621                        25                                 16.3   
14185                       142                                 78.2   

               Provider Type                             Legal Business Name  \
148    Medicare and Medicaid               LAUDERDALE CHRISTIAN NURSING HOME   
190    Medicare and Medicaid                       PINES BIRMINGHAM OPCO LLC   
219                 Medicaid               Legal Business Name Not Available   
345                 Medicare                    SUN HEALTH GRANDVIEW TERRACE   
672    Medicare and Medicaid  PROVIDENCE HEALTH SYSTEM - SOUTHERN CALIFORNIA   
...                      ...                                             ...   
13398  Medicare and Medicaid              VILLA SCALABRINI RETIREMENT CENTER   
13453  Medicare and Medicaid                        BETHEL LUTHERAN HOME INC   
13550  Medicare and Medicaid              MCCULLOUGH HALL NURSING CENTER INC   
13621               Medicare                          THE METHODIST HOSPITAL   
14185  Medicare and Medicaid               SOUTH LIMESTONE HOSPITAL DISTRICT   

       ... Rating Cycle 2 Standard Health Survey Date  \
148    ...                                  7/25/2019   
190    ...                                  9/26/2018   
219    ...                                 12/14/2019   
345    ...                                  4/21/2023   
672    ...                                 10/15/2023   
...    ...                          

In [87]:
church_nh.groupby(['State', 'Ownership Type'])['Overall Rating'].mean()

State  Ownership Type             
AL     Non profit - Church related    2.333333
AZ     Non profit - Church related    4.000000
CA     Non profit - Church related    4.076923
CO     Non profit - Church related    4.000000
CT     Non profit - Church related    4.000000
DC     Non profit - Church related    5.000000
FL     Non profit - Church related    3.750000
GA     Non profit - Church related    2.750000
IA     Non profit - Church related    3.062500
IL     Non profit - Church related    4.160000
IN     Non profit - Church related    2.888889
KS     Non profit - Church related    3.714286
KY     Non profit - Church related    4.000000
LA     Non profit - Church related    2.333333
MA     Non profit - Church related    3.000000
MD     Non profit - Church related    4.166667
ME     Non profit - Church related    1.000000
MI     Non profit - Church related    4.000000
MN     Non profit - Church related    3.066667
MO     Non profit - Church related    3.714286
NC     Non profit - Churc

In [88]:
church_nh.groupby(['Ownership Type'])['Overall Rating'].mean()

Ownership Type
Non profit - Church related    3.522491
Name: Overall Rating, dtype: float64

In [89]:
corp_np_nh=state_non_profit[state_non_profit['Ownership Type']=='Non profit - Corporation']
corp_np_nh

Processing Date                                   Provider Name  \
18           1/1/2025                        EAMC LANIER NURSING HOME   
21           1/1/2025  EASTERN SHORE REHABILITATION AND HEALTH CENTER   
32           1/1/2025                                      FAIR HAVEN   
38           1/1/2025                 ALLEN HEALTH AND REHABILITATION   
62           1/1/2025                        JOHN KNOX MANOR INC  I I   
...               ...                                             ...   
14370        1/1/2025                         THE LEGACY MIDTOWN PARK   
14376        1/1/2025      STARR COUNTY NURSING AND TRANSITIONAL CARE   
14383        1/1/2025                             ADVENTHEALTH DELAND   
14393        1/1/2025           JAMES L WEST CENTER FOR DEMENTIA CARE   
14395        1/1/2025                       WARE MEMORIAL CARE CENTER   

                  City State      County            Ownership Type  \
18              VALLEY    AL    Chambers  Non profit - Corporation   
21              DAPHNE    AL     Baldwin  Non profit - Corporation   
32          BIRMINGHAM    AL   Jefferson  Non profit - Corporation   
38              MOBILE    AL      Mobile  Non profit - Corporation   
62          MONTGOMERY    AL  Montgomery  Non profit - Corporation   
...                ...   ...         ...                       ...   
14370           DALLAS    TX    Anderson  Non profit - Corporation   
14376  RIO GRANDE CITY    TX       Starr  Non profit - Corporation   
14383           DELAND    FL     Volusia  Non profit - Corporation   
14393       FORT WORTH    TX     Tarrant  Non profit - Corporation   
14395         AMARILLO    TX      Potter  Non profit - Corporation   

       Number of Certified Beds  Average Number of Residents per Day  \
18                          103                                 72.6   
21                          117                                 99.2   
32                          259                                234.4   
38                          119                                 78.6   
62                           98                                 61.2   
...                         ...                                  ...   
14370                        54                                 48.3   
14376                       120                                102.4   
14383                        18                                 18.3   
14393                       112                                104.3   
14395                       120                                 86.4   

               Provider Type                            Legal Business Name  \
18     Medicare and Medicaid         THE EAST ALABAMA HEALTH CARE AUTHORITY   
21     Medicare and Medicaid                       NOLAND EASTERN SHORE LLC   
32     Medicare and Medicaid                   METHODIST HOME FOR THE AGING   
38     Medicare and Medicaid                NOLAND MANAGEMENT SERVICES, LLC   
62     Medicare and Medicaid                         JOHN KNOX MANOR INC II   
...                      ...                                            ...   
14370  Medicare and Medicaid                    THE LEGACY MIDTOWN PARK INC   
14376  Medicare and Medicaid                         OAKBEND MEDICAL CENTER   
14383  Medicare and Medicaid             MEMORIAL HOSPITAL-WEST VOLUSIA INC   
14393               Medicare  JAMES L WEST PRESBYTERIAN SPECIAL CARE CENTER   
14395  Medicare and Medicaid                     BAPTIST COMMUNITY SERVICES   

       ... Rating Cycle 2 Standard Health Survey Date  \
18     ...                                  3/21/2019   
21     ...                                  1/31/2019   
32     ...                                  1/30/2020   
38     ...                                  1/31/2019   
62     ...                                 10/10/2019   
...    ...                                        ...   
14370  ...                                  7/20/2023   
14376  ...        

In [90]:
corp_np_nh.groupby(['State', 'Ownership Type'])['Overall Rating'].mean()

State  Ownership Type          
AK     Non profit - Corporation    3.900000
AL     Non profit - Corporation    2.947368
AR     Non profit - Corporation    2.583333
AZ     Non profit - Corporation    2.909091
CA     Non profit - Corporation    3.959596
CO     Non profit - Corporation    3.562500
CT     Non profit - Corporation    3.656250
DC     Non profit - Corporation    4.375000
DE     Non profit - Corporation    4.111111
FL     Non profit - Corporation    3.456140
GA     Non profit - Corporation    2.931818
HI     Non profit - Corporation    4.333333
IA     Non profit - Corporation    3.136364
ID     Non profit - Corporation    3.000000
IL     Non profit - Corporation    3.587500
IN     Non profit - Corporation    3.141026
KS     Non profit - Corporation    3.172840
KY     Non profit - Corporation    3.184211
LA     Non profit - Corporation    3.000000
MA     Non profit - Corporation    3.679487
MD     Non profit - Corporation    3.800000
ME     Non profit - Corporation    3.562500


In [91]:
corp_np_nh.groupby(['Ownership Type'])['Overall Rating'].mean()

Ownership Type
Non profit - Corporation    3.440727
Name: Overall Rating, dtype: float64

In [92]:
nursing_homes['Ownership Type']== 'Government - Federal'

0        False
1        False
2        False
3        False
4        False
         ...  
14398    False
14399    False
14400    False
14401    False
14402    False
Name: Ownership Type, Length: 14403, dtype: bool

In [93]:
nursing_homes['Ownership Type'].unique()

array(['For Profit', 'Government- City/County',
       'Non profit - Corporation', 'Non profit - Church related',
       'Government - Federal', 'Government - State',
       'Government - Hospital district'], dtype=object)

In [94]:
va_nh=nursing_homes[nursing_homes['Ownership Type']== 'Government - Federal']

In [95]:
va_nh.groupby(['State', 'Ownership Type'])['Overall Rating'].mean()

State  Ownership Type      
AZ     Government - Federal    2.0
CA     Government - Federal    2.0
FL     Government - Federal    4.0
MS     Government - Federal    1.5
NE     Government - Federal    5.0
OH     Government - Federal    1.0
SC     Government - Federal    3.0
TN     Government - Federal    3.5
VA     Government - Federal    1.0
WA     Government - Federal    4.0
WY     Government - Federal    3.0
Name: Overall Rating, dtype: float64

In [96]:
va_nh[va_nh['State']=='AZ']

Processing Date                                     Provider Name  \
322        1/1/2025                                      CARING HOUSE   
343        1/1/2025  ARCHIE HENDRICKS SENIOR SKILLED NURSING FACILITY   

        City State County        Ownership Type  Number of Certified Beds  \
322  SACATON    AZ  Pinal  Government - Federal                       100   
343    SELLS    AZ   Pima  Government - Federal                        60   

     Average Number of Residents per Day          Provider Type  \
322                                 86.1  Medicare and Medicaid   
343                                 45.7  Medicare and Medicaid   

                              Legal Business Name  ...  \
322                              THE CARING HOUSE  ...   
343  ARCHIE HENDRICKS SR SKILLED NURSING FACILITY  ...   

    Rating Cycle 2 Total Health Score  \
322                              28.0   
343                              60.0   

    Rating Cycle 3 Standard Health Survey Date  \
322                                   9/2/2021   
343                                  8/27/2021   

    Rating Cycle 3 Total Health Score Number of Facility Reported Incidents  \
322                              36.0                                     1   
343                              32.0                                     6   

    Number of Substantiated Complaints  \
322                                  1   
343                                  0   

    Number of Citations from Infection Control Inspections Number of Fines  \
322                                                NaN                   0   
343                                                NaN                   1   

     Total Amount of Fines in Dollars  Total Number of Penalties   Total  
322                              0.00                          0    0.00  
343                            662.73                          1  662.73  

[2 rows x 62 columns]

In [97]:
nursing_homes

Processing Date                                      Provider Name  \
0            1/1/2025                           BURNS NURSING HOME, INC.   
1            1/1/2025                     COOSA VALLEY HEALTHCARE CENTER   
2            1/1/2025                         HIGHLANDS HEALTH AND REHAB   
3            1/1/2025        EASTVIEW REHABILITATION & HEALTHCARE CENTER   
4            1/1/2025                      PLANTATION MANOR NURSING HOME   
...               ...                                                ...   
14398        1/1/2025  LAS ALTURAS NURSING & TRANSITIONAL CARE BROWNS...   
14399        1/1/2025        RIVERSIDE NURSING AND REHABILITATION CENTER   
14400        1/1/2025  FIVE POINTS NURSING & REHABILITATION OF COLLEG...   
14401        1/1/2025                   HCA HOUSTON HEALTHCARE SOUTHEAST   
14402        1/1/2025                       LEGACY ESTATE LONG TERM CARE   

                  City State     County                  Ownership Type  \
0         RUSSELLVILLE    AL   Franklin                      For Profit   
1            SYLACAUGA    AL  Talladega                      For Profit   
2           SCOTTSBORO    AL    Jackson         Government- City/County   
3           BIRMINGHAM    AL  Jefferson                      For Profit   
4             MC CALLA    AL  Jefferson                      For Profit   
...                ...   ...        ...                             ...   
14398      BROWNSVILLE    TX    Cameron                      For Profit   
14399        KERRVILLE    TX       Kerr                      For Profit   
14400  COLLEGE STATION    TX     Brazos                      For Profit   
14401         PASADENA    TX     Harris                      For Profit   
14402         COMANCHE    TX   Comanche  Government - Hospital district   

       Number of Certified Beds  Average Number of Residents per Day  \
0                            57                                 53.5   
1                            85                                 76.4   
2                            50                                 45.7   
3                            92                                 69.3   
4                           103                                 87.6   
...                         ...                                  ...   
14398                        90                                  NaN   
14399                       100                                 31.0   
14400                       116                                  NaN   
14401                         4                                  6.3   
14402                       100                                  NaN   

               Provider Type                             Legal Business Name  \
0      Medicare and Medicaid                        BURNS NURSING HOME, INC.   
1      Medicare and Medicaid              COOSA VALLEY HEALTHCARE CENTER LLC   
2      Medicare and Medicaid                  HH HEALTH SYSTEM - JACKSON LLC   
3      Medicare and Medicaid                    BALL HEALTHCARE EASTVIEW INC   
4      Medicare and Medicaid                 C & G HEALTHCARE SERVICES, INC.   
...                      ...                                             ...   
14398  Medicare and Medicaid     HERITAGE BROWNSVILLE HEALTH DEVELOPMENT LLC   
14399  Medicare and Medicaid             KERRVILLE HEALTHCARE OPERATIONS LLC   
14400  Medicare and Medicaid               Legal Business Name Not Available   
14401               Medicare                                CHCA BAYSHORE LP   
14402  Medicare and Medicaid  COMANCHE COUNTY CONSOLIDATED HOSPITAL DISTRICT   

       ... Rating Cycle 2 Total Health Score  \
0      ...                               8.0   
1      ...                               4.0   
2      ...                              20.0   
3      ...                               4.0   
4      ...                               4.0   
...    ...                               ...   
14398  ...                    

In [101]:
al_nh=nursing_homes[nursing_homes['State']=='AL']

In [109]:
al_nh[['Staffing Rating']].mean()

Staffing Rating    3.354545
dtype: float64